src/System.Runtime/src/ApiCompatBaseline.txt

```diff
@@ -0,0 +1,5 @@
+#DebuggableAttribute members need to be ported to System.Private.Corelib in .NETNative (https://github.com/dotnet/corefx/issues/9699) :
```

Comment:
Can you please rename to ApiCompatBaseline.netcore50aot.txt so it only applies to that configuration.